sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md

```diff
@@ -1,14 +1,22 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2023-12-20)
```

Comment:
since the date has passed, could you update the date here as well?

Reply:
updated